src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,154 @@
+* Fix #25022: `az resource tag`: Fix updating tag for Microsoft.insights/workbooks issue (#25142)
+
+**ARO**
+
+* `az aro create/update`: Add NetworkContributor role to NAT Gateways in Cluster Resource Group when creating or updating clusters (#25211)
+* `az aro create`: Change `--pull-secret` parameter to no longer require `@` prefix on filenames (#25211)
+
+**Backup**
+
+* `az backup vault`: Add new parameter `--public-network-access` to support enabling public network access for the backup vault (#24699)
+* `az backup vault create`: Add new parameter `--immutability-state` to support configuring immutability settings for the backup vault (#25277)
+
+**Batch**
+
+* Fix: `az batch pool create`: Fix bug that caused error when parameter `--encryption-key-identifier` is used (#25034)
+* Test recording cleanup (#25166)
+
+**Code**
+
+* `aaz`: Support `AAZListArg` singular options list (#25099)
+
+**Compute**
+
+* `az image builder identity assign`: Add this command to add managed identity to an existing image builder template (#25136)
+* `az image builder identity remove`: Add this command to remove managed identity from an existing image builder template (#25136)
+* `az image builder identity show`: Add this command to display managed identity of a image builder template (#25136)
+* `az vmss reimage`: Let `--instance-id` support multiple ids (#25131)
+* Fix #25308: `az disk create`: Fix help message for creating a standard disk for uploading blobs (#25315)
+* `az vmss create/update`: Add new parameter `--enable-osimage-notification` to support enabling OS image scheduled event (#25273)
+* `az vmss create`: Add new parameter `--max-surge` to support enabling rolling upgrade policy max surge (#25333)
+
+**Cosmos DB**
+
+* `az managed-cassandra datacenter update`: Add support to update `--sku` (#25215)
+
+**DMS**
+
+* `az dms project task create`: Update DMS MySQL API to support new migration types (#24978)
+* Change Azure Database Migration Service string to Azure Database Migration Service (classic) (#25258)
+
+**Feedback**
+
+* Stop including error messages in the feedback body (#25286)
+
+**IoT**
+
+* `az iot hub wait`: Add wait commands (#25326)
+* `az iot hub delete`: Fix functionality issue for parameter `--no-wait` (#25326)
+
+**Key Vault**
+
+* `az keyvault security-domain restore-blob`: Support restoring blob offline (#25270)
+* `az keyvault security-domain upload`: Add `--restore-blob` to prevent exposing keys in online environment (#25270)
+
+**NetAppFiles**
+
+* Fix volume patch dataprotection props (#25231)
+
+**Network**
+
+* [BREAKING CHANGE] `az network application-gateway ssl-profile`: Output property `verifyClientCertIssuerDn` is renamed by `verifyClientCertIssuerDN` to keep consistent with the name in API (#25143)
+* [BREAKING CHANGE] `az network cross-region-lb frontended-ip`: Rename output properties with `publicIp` prefix by `publicIP` prefix to keep consistent with the names in API response (#25157)
```

Comment:
"Rename" should be followed by `to`, not `by`: https://english.stackexchange.com/questions/100296/which-preposition-to-use-with-rename